src/pages/index.js

```diff
@@ -29,14 +29,14 @@ const Home = () => (
       <ResourceCard
         lead="Experience VR"
         title="Firefox"
-        desc="Experience WebVR with Firefox; Mozilla's fast, private and ethical web browser."
+        desc="Experience WebVR with Firefox: Mozilla's fast, private, and ethical web browser."
```

Comment:
semicolon to colon change was the primary change here. I added the Oxford comma. I assume you approve of that change, as an Oxford comma advocate :wink:

Reply:
I am ashamed. Approved:)